src/api/services/participantsService.ts

```diff
@@ -0,0 +1,31 @@
+import { z } from 'zod';
+
+import { ParticipantSchema } from '../entities/Participant';
+import { ParticipantType } from '../entities/ParticipantType';
+import { SSP_TAM_EMAIL, SSP_TAM_EMAIL_DISPLAY_NAME, SSP_WEB_BASE_URL } from '../envars';
+import { createEmailService } from './emailService';
+import { EmailArgs } from './emailTypes';
+
+export const sendNewParticipantEmail = async (
+  newParticipant: z.infer<typeof ParticipantSchema>,
+  participantTypes: ParticipantType[]
+) => {
+  const emailService = createEmailService();
+  const requestor = newParticipant.users![0];
+  const templateData = {
+    participant: newParticipant.name,
+    participantType: participantTypes.map((pt) => pt.typeName).join(', '),
+    requestor: `${requestor.firstName} ${requestor.lastName}`,
+    requestorEmail: requestor.email,
+    participantLocation: requestor.location,
+    jobFunction: requestor.role,
+    link: SSP_WEB_BASE_URL,
+  };
+  const emailArgs: EmailArgs = {
+    subject: 'New Participant Request',
+    templateData,
+    template: 'newParticipantReadyForReview',
+    to: { name: SSP_TAM_EMAIL_DISPLAY_NAME, email: SSP_TAM_EMAIL },
+  };
+  emailService.sendEmail(emailArgs);
```

Comment:
should we catch any errors here, and send back a standardised error code - so that the caller can distinguish between email errors vs creational/querying errors?

Reply:
I think this is a good idea. I will do this now. I think if the email does not go through we shouldn't block the user , am I right?

Reply:
I think so, Maybe we can ask Kimberly on this as well.

IMO From the user's perspective they haven't done anything wrong, and if they repeat - will our database even let them create again ? If they can't create again, then better for us to alert ourselves and take a manual reconciliation.